src/encoded/static/components/award.js

```diff
@@ -1687,8 +1687,8 @@ MilestonesTable.propTypes = {
  */
 const fillDates = (sortedDateTerms, startDate) => {
     // The new array limits go between `startDate` and the current date.
-    const startDateMoment = moment(startDate, 'YYYY-MM');
-    const endDateMoment = moment();
+    const startDateMoment = dayjs(startDate, 'YYYY-MM');
+    const endDateMoment = dayjs();
```

Comment:
I did not think so because "Moment" is still a word for time. But, since you see a problems, I can see how others may see a problem too. I will update the variable names.

Reply:
Done.